test/jasmine/tests/plots_test.js

```diff
@@ -326,7 +383,7 @@ describe('Test Plots', function() {
         beforeEach(function(done) {
             gd = createGraphDiv();
 
-            Plotly.plot(gd, [{ x: [1, 2, 3], y: [2, 3, 4] }], {})
+            Plotly.plot(gd, [{ x: [1, 2, 3], y: [2, 3, 4] }], { autosize: true })
```

Comment:
@n-riesco , @chriddyp  and I are testing the `mapbox` branch (which is rebased on top of the latest `master`) in the plot.ly workspace at the moment and we some issues.

Looks like this PR here is backward incompatible and the line :arrow_double_up: is a symptom. 

More clearly,

```js
Plotly.plot(gd, data);
```

should go through the `plotAutoSize` routine on the initial `Plotly.plot` call.

I believe changing the `autosize` attribute `dflt` to `true` would solve this. What are your thoughts on that?